modules/rosa-getting-started-install-configure-cli-tools.adoc

```diff
@@ -50,7 +50,7 @@ You can alternatively use the `AWS_DEFAULT_REGION` environment variable to set t
 +
 [source,terminal]
 ----
-$ aws sts get-caller-identity
+$ aws sts get-caller-identity  --output text
```

Comment:
There are two spaces before the flag here. Seeing as this won't affect the command though, I won't hold up the merge over it.